app/controllers/unattended_controller.rb

```diff
@@ -182,7 +182,7 @@ def set_content_type
 
   def load_template_vars
     # load the os family default variables
-    send "#{@host.os.pxe_type}_attributes"
+    send "#{@host.operatingsystem.pxe_type}_attributes"
```

Comment:
Many files use `@host.os` and `@host.operatingsystem`. I would prefer to use only one alias/attribute per file to make it cleaner.

What your thoughts about this?